frame/vesting/src/lib.rs

```diff
@@ -648,6 +653,34 @@ impl<T: Config> Pallet<T> {
 
 		Ok((schedules, locked_now))
 	}
+
+	/// Ensure the correctness of the state of this pallet.
+	///
+	/// This should be valid before or after each state transition of this pallet.
+	///
+	/// ## Invariants:
+	///
+	/// * The `per_block` payments left of every account currently vesting
+	/// * is equal to the total balance currently locked.
+	#[cfg(any(feature = "try-runtime", test))]
+	pub fn do_try_state() -> Result<(), &'static str> {
+		Vesting::<T>::iter().for_each(|(_, d)| {
+			let vesting = d.to_vec();
+			let mut total_per_block: BalanceOf<T> = Zero::zero();
+			let mut total_locked_now: BalanceOf<T> = Zero::zero();
+			for info in vesting.iter() {
+				// get the number of remaining vesting blocks<>balance
+				let payments_left: BalanceOf<T> =
+					info.ending_block_as_balance::<T::BlockNumberToBalance>();
```

Comment:
I'm confused by this variable name. `ending_block_as_balance` returns "Block number at which the schedule ends", it doesn't seem analogous to `payments_left` to me.

Reply:
`amount_left` seems a better fit?